bootcamp-milestone-2/src/app/blogs/page.tsx

```diff
@@ -0,0 +1,51 @@
+import Image from "next/image";
+import styles from "./page.module.css";
+import Link from "next/link";
+import blogs from "@/blogData";
+import BlogPreview from "../components/blogPreview";
+// import blogs from '@/blogData';
+// import BlogPreview from '../components/blogPreview';
+// import blogs from "blogData.ts";
+// import BlogPreview from "@/components/blogPreview";
+
+export default function Blogs() {
+  <Link href="/page.tsx"></Link>;
+  return (
+    <>
+      <nav className="navbar">
```

Comment:
You don't need the navbar anymore since you already shouldve made a component and added it to your layout.tsx